composer.json

```diff
@@ -3,17 +3,21 @@
   "description": "Prometheus Push Gateway client for PHP applications.",
   "type": "library",
   "require": {
-    "php": "^7.2|^8.0",
-    "guzzlehttp/guzzle": "^6.3|^7.0",
-    "promphp/prometheus_client_php": "^1.0|^2.0"
+    "php": "^7.2||^8.0",
+    "promphp/prometheus_client_php": "^1.0||^2.0",
+    "symfony/http-client": "^5.2"
```

Comment:
Maybe I am still a little fuzzy on the details here, but did we just replace a Guzzle dependency with a Symfony Http Client dependency instead?

Reply:
No we are not. The symfony Http Client is used as a proof that the PSR abstraction can be use with any PSR compliant HTTP Client. Where I agree with you is that it could be removed from the `composer.json` suggest section. Also if needed I could remove completely the `SymfonyFactory` class all together since there's a `PsrFactory` class. But it is still needed in the `require-dev` section.